jwst/outlier_detection/outlier_detection_step.py

```diff
@@ -61,6 +63,7 @@ class OutlierDetectionStep(Step):
         backg = float(default=0.0)
         kernel_size = string(default='7 7')
         threshold_percent = float(default=99.8)
+        n_ints = integer(default=25)
```

Comment:
I know that the units of this param value are the number of integrations, but I wonder if we could figure out a param name that makes it a bit more obvious as to how it gets used. Something along the lines of `rolling_median_size`? Other suggestions are welcome.

Reply:
changed to `rolling_window_width`